packages/experimental/Avatar/src/useAvatar.ts

```diff
@@ -41,7 +42,7 @@ export const useAvatar = (props: JSAvatarProps): AvatarInfo => {
     showBadge,
   };
 
-  const _initials = initials || getInitials(name);
+  const _initials = initials || getInitials(name, I18nManager.isRTL);
```

Comment:
I don't think you'll need an isRTL flag - RTL strings still have the same "order" they just render right to left instead of left to right. So the 0th character is right most in RTL, like the 0th character is the left most in LTR.